bears/general/CustomExceptions.py

```diff
@@ -0,0 +1,11 @@
+class UnmatchedIndentError(Exception):
```

Comment:
same as for the other fine i'll move it in another commit on the coala repository? but it'lll have to be on a differnet PR?